lib/cfa/augeas_parser.rb

```diff
@@ -217,7 +217,7 @@ def []=(key, value)
     # @param matcher [Matcher]
     # @return [Array<AugeasElement>] matching elements
     def select(matcher)
-      @data.select(&matcher)
+      data.select(&matcher)
```

Comment:
Before this fix, this line is one of the four lines in this file not covered by tests :-)